res/res_pjsip_outbound_registration.c

```diff
@@ -743,6 +750,26 @@ static pj_status_t registration_client_send(struct sip_outbound_registration_cli
 	/* Add Security-Verify or Security-Client headers */
 	add_security_headers(client_state, tdata);
 
+	/*
```

Comment:
Nitpick on coding guidelines, but the * should be indented one space after the first one, so that the stars all line up

Reply:
Ohh, thanks! Fixed!